v2.0/releases/index.md

```diff
@@ -6,6 +6,40 @@ The following table shows component versioning for Calico  **{{ page.version }}*
 
 Use the version selector at the top-right of this page to view a different release.
 
+
+## v2.0.1 - Release Notes
+
+This is a bug fix release, the major fixes and enhancements include:
+- Fix: some policies not working because of / in the labels.
+- Fix that ipv4_nat was not working (for OpenStack floating IPs)
+- Fix logic in static link test. Move inside container.
```

Comment:
Incomprehensible IMO.  In the Felix release notes I squashed this into "Fix slow Felix builds".